Medbay remap

comma opened the pull request:
Flipped it horizontaly, so treatment area is right next to the entrance and second OR.
Rearranged things a bit to increase visibiliy between room, e.g. now can sit in exam room anad see seats of lobby desk, so participate in chatter.
Since some space freed up, expanded morgue into tumorous luxury suite.
Made robotics OR its own area reee.

![](https://vgy.me/U2b5cJ.png)


<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

ghost commented:
I kinda like it as-is. I like having the main care room not viewable from the the lobby, to keep people from seeing all the ~~dead bodies~~ healing. I do like the robotics OR getting its own area, though.

babydoll commented:
Yeah, for a hospital there's not a lot of privacy for basically anything

comma commented:
It's not a hospital, it's an infirmary. You don't get privacy. You're there to be fixed up and shipped out.
The Eternal Walk from lobby to actual work area, and from scanner to 2nd OR is worse than 'privacy' issues IMO.

Chiquiii commented:
There's a fire alarm on a window of the sub-acute ward.  The window tint button in the exam room will overlap with the fire alarm from the lobby. And please don't place buttons or other objects on windows. Pixel shift the button to open the treatment center door onto the wall to its right.

artemsablin commented:
@comma Any way to place the operating table in the middle of the room as opposed to sticking it to the wall? 

Additionally, the OR could be expanded a bit with the area substracted from the acute ward, with a buffer "prep area" added.

HeyBanditoz commented:
It was _just_ remapped, and now it's being remapped again?

ghost commented:
I think the recent remap was sufficient, In my opinion. The only I would like to see changed is the size of the OR, as it can get a little cramped when you have multiple surgeons working, but then again, this is the Torch, not a specialized hospital ship.

ghost commented:
@rf-artemsablin If that was done, we'd also need to either expand the range of the wall mounted Sleeping gas machine, or just nix it.

artemsablin commented:
@SamSamsonthe3rd it's more or less fine. I have to point out that in the pictures it looks like the morgue has a decent operating room, whereas in the normal one the surgeons are often times forced to hop onto the tables to do something.

In the IRL ORs I've worked in the operating table was in the middle with ample space around it for the necessary toolstands and for everyone needed to fit in. I tried google searching images of ORs around the world and this seems to be the same case everywhere. Don't see any justification for not placing it in the middle, both common-sense and gameplay-wise.

Hubblenaut commented:
I prefer the current one, as now, equipment storage, chemistry and morgue are annoyingly far away from the treatment-area and the examination room is no longer on a direct way toward it.

Also, lone-standing lockers are easily pushed on accident and a big :-1: IMO. Should avoid those.

IDTia commented:
I really like this remap, a huge 👍 for me.

Issues I see is the recovery room has no wire to it's APC.

Chronographology commented:
I personally like this design, it's a better use of space. We don't have the, rather blank, empty hallways from the last redesign anymore and the ORs look much better. The double fridges justifies the longer walk to Chemistry, as long as it gets stocked. A big 👍 from me, personally.

dore0062 commented:
The only weird thing is the ladder in the middle of the general area

IDTia commented:
The locker room is missing a camera, now that it's a bigger room, if you want one in there or not, the APC wire again in treatment, the nanomed above the sleeper is not reachable, the old layout had a newscaster above the sleeper that was not reachable too.

comma commented:
> Don't see any justification for not placing it in the middle, both common-sense and gameplay-wise.

Wall mounted anesthesia tanks, that's p much it. Morgue patients are usually fine without anesthesia, that's why I was able to put it in the middle like that, since autopsies can deal with more gawkers.

>can get a little cramped when you have multiple surgeons working
> the surgeons are often times forced to hop onto the tables to do something.

I didn't change OR aside from flipping it around, you have /3/ tiles to stand near OR. If you want more than 3 people working on same person for some reason, that is indeed not normal, and you can deal with having to hop on table. 

>The locker room is missing a camera

It didn't have one, tbh I'd like to keep it that way for PRIVACY.

>The only weird thing is the ladder in the middle of the general area

Can't really move bottom storage room, and it's not quite in the middle, it's in the corner.

>equipment storage, chemistry and morgue are annoyingly far away from the treatment-area

Equipment storage isn't that far, there's a fridge for meds in treatment area you can stock with most used ones, and other two aren't that important time-of-travel wise to justify actual /treatment/ area being that far from the entrance.

IDTia commented:
Could put Windoors and glass on the ladders, because it's sort of a room with sensitive stuff in it, a syringe gun, straight jacket and what not. That, or put the storage stuff behind glass as well.

comma commented:
Added dem windows

comma commented:
Something's afoot with area coherency test.
Passing test: 
![](https://vgy.me/DfnJGP.png)
Failing test:
![](https://vgy.me/5Fujlk.png)

comma commented:
Nevermind I'm an idiot, there were space tiles under those windowspawners.

ghost commented:
I notice the main scanner in the screenshot above has a sort of "D" shaped red tile pattern around it. Could you change that to a more circular pattern, so the OCD of every medical player doesn't flip out?

artemsablin commented:
@comma 
>Wall mounted anesthesia tanks, that's p much it. 

Would strongly advise at least moving the steel table with a surgery kit somewhere else to allow two surgeons access the patient without table-jumping (which is what occurs now).

IDTia commented:
Keep in mind you can walk through IVs now, three people can crowd around an operating table, in the remap we have now table jumping has already stopped because we have three spots to stand.

comma commented:
>Could you change that to a more circular pattern, so the OCD of every medical player doesn't flip out?

Right, removing lower corner of it then :3

Also yeah what cat said, you can have 3 people working on the guy if for some reason you need it. Forr /4/ you can tablehop holy fuck what are you doing.

Hubblenaut commented:
Minor nitpick, the paper bins all look like they are about to fall off the table :v
They are usually mapped with a slight pixelshift.

**Edit:** Yet another very minor nitpick, the APC in the locker room might be better off on the free spot on the wall rather than behind the locker.

Also, not sure what to think about surrounding the ladder with windows... looked better how it was before IMO, the windows make that corner of the room look very cramped.

ghost commented:
![image](https://user-images.githubusercontent.com/31956636/32720441-d7a79062-c863-11e7-914d-2af3cbbc2e5f.png)
shouldn't there be a wall where skeletor is standing to support the emergency shutter?

dryerlint commented:
If the ladder in the Treatment Center looks too weird, maybe you could get rid of it completely, and just use the one in the maintenance area to the left? You could remap that small area into an "upper medical storage room" or whatever, and add a door blocking it off from the maint tunnels

comma commented:
It's there because I can't move room below, moving only upper ladder wouldn't really work very well.

comma commented:
Splicer where incubator is
Analyser where centrifuge is
Incubator where isolator is
Isolator where analyser is
Centrifuge where splicer is

comma commented:
Unholy diff is because map key length changed, can't do anything about that.

comma committed:
Medbay remap yes again

Flipped it horizontaly, so treatment area is right next to the entrance and second OR. Rearranged things a bit to increase visibiliy between room, e.g. now can sit in exam room anad see seats of lobby desk, so participate in chatter. Since some space freed up, expanded morgue into tumorous luxury suite. Made robotics OR its own area reee.

PurpleMartinJCK reviewed (approved):
Looks fine to me.

PurpleMartinJCK commented:
I kinda like it but also kind of don't. We'll have to see how it plays out. At least getting new organs will be way faster.
